src/content/en/2020/privacy.md

```diff
@@ -12,12 +12,166 @@ translators: []
+{{ figure_markup(
+  image="adoption_of_the_TCF_banner.png",
+  caption="Adoption rate of TCF banner",
+  description="This figure shows the percentage of websites using IAB Europe's TCF banner.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=341275612&format=interactive",
+  sheets_gid="2077755325",
+    sql_file="percent_of_websites_with_iab_tcf_banner.sql"
+  )
+}}
+While our results show that the TCF banner is not yet the ‘industry standard’, it is a step in the right direction. Considering the main target group of IAB Europe is in fact European publishers, having an adoption rate on 1.5% of websites on desktop client and 1.4% on mobile is not too bad. 
+
+## Privacy Policies
+Privacy policies are widely used by websites to increase transparency towards users about data collection practices. In our crawl, we searched for keywords indicating the presence of a privacy policy text on each visited website. 
+
+{{ figure_markup(
+  image="websites_with_privacy_link.png",
+  caption="Websites that have a privacy policy",
+  description="This figure shows the percentage of websites that indicate to have implemented a privacy policy text.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=329249623&format=interactive",
+  sheets_gid="495362514",
+    sql_file="percent_of_websites_with_privacy_links.sql "
+  )
+}}
+The results show that almost half of the websites in the dataset have included a privacy policy, which is positive. However, studies have shown that the majority of internet users do not bother reading privacy policies and when they do, they lack understanding due to the length and complexity of most privacy policy texts.
+
+## Conclusion
+This chapter gives an overview of the current state of privacy on the web. Third-party tracking remains prominent on both desktop and mobile clients, with Google tracking the largest percentage of websites. Consent management platforms are used on a small percentage of websites, however a lot of websites implement their own cookie consent banners. Lastly, roughly half of the websites include a privacy policy, which benefits greatly transparency towards users about data processing practices.  This is undoubtedly a step forward but there is a lot to be done. Most of the time, privacy policies are hard to read and understand and cookie consent banners manipulate users into consent. For the web to truly respect users, privacy has to be a part of conception, not an afterthought. Regulations are a good thing and it is reassuring to see them appearing worldwide but, more than that and financial sanctions, Privacy by Design should be the norm, rather than deploying texts and tools in order to avoid fines.
```

Comment:
```suggestion
This chapter gives an overview of the current state of privacy on the web. Third-party tracking remains prominent on both desktop and mobile clients, with Google tracking the largest percentage of websites. Consent Management Platforms are used on a small percentage of websites, however a lot of websites implement their own cookie consent banners. Lastly, roughly half of the websites include a privacy policy, which benefits greatly transparency towards users about data processing practices. This is undoubtedly a step forward but there is a lot to be done. Often, privacy policies are hard to read and understand and cookie consent banners manipulate users into consent. For the web to truly respect users, privacy has to be a part of conception, not an afterthought. Regulation is a good thing and it is reassuring to see an increase in privacy regulation worldwide but, more than that and financial sanctions, [privacy by design](https://en.wikipedia.org/wiki/Privacy_by_design) should be the norm, rather than deploying policies and tools in order to avoid fines.
```